bmv2/Makefile.am

```diff
@@ -29,6 +29,35 @@ $(builddir)/@P4_NAME@/thrift-src/pd_rpc_server.h
 
 P4_JSON_OUTPUT = $(P4_NAME).json
 
+plugin_files =
+plugin_headers =
+
+# Openflow plugin
+if COND_PLUGIN_OPENFLOW
+of_sources = \
+@P4_NAME@/plugin/of/src/pd_wrappers.c    \
+@P4_NAME@/plugin/of/src/ofpat_groups.c   \
+@P4_NAME@/plugin/of/src/ofpat_state.c    \
+@P4_NAME@/plugin/of/src/ofpat_pipeline.c
+
+of_includes = \
+@P4_NAME@/plugin/of/inc/pd_wrappers.h    \
+@P4_NAME@/plugin/of/inc/ofpat_groups.h   \
+@P4_NAME@/plugin/of/src/ofpat_state.h    \
+@P4_NAME@/plugin/of/inc/ofpat_pipeline.h \
+@P4_NAME@/plugin/of/inc/openflow.h
+
+plugin_files += $(of_sources) $(of_includes)
+plugin_headers += $(of_includes)
```

Comment:
They're not needed anymore by p4ofagent since the addition of the
"plugin.h" header in p4ofagent, which adds some extern declarations and a
typedef for things p4ofagent depended on.

On Thu, Jun 16, 2016 at 10:09 AM, Antonin Bas <notifications@github.com>
wrote:

> In bmv2/Makefile.am
> <https://github.com/p4lang/p4-build/pull/5#discussion_r67384920>:
>
> > +if COND_PLUGIN_OPENFLOW
> > +of_sources = \
> > +@P4_NAME@/plugin/of/src/pd_wrappers.c    \
> > +@P4_NAME@/plugin/of/src/ofpat_groups.c   \
> > +@P4_NAME@/plugin/of/src/ofpat_state.c    \
> > +@P4_NAME@/plugin/of/src/ofpat_pipeline.c
> > +
> > +of_includes = \
> > +@P4_NAME@/plugin/of/inc/pd_wrappers.h    \
> > +@P4_NAME@/plugin/of/inc/ofpat_groups.h   \
> > +@P4_NAME@/plugin/of/src/ofpat_state.h    \
> > +@P4_NAME@/plugin/of/inc/ofpat_pipeline.h \
> > +@P4_NAME@/plugin/of/inc/openflow.h
> > +
> > +plugin_files += $(of_sources) $(of_includes)
> > +plugin_headers += $(of_includes)
>
> Btw, I am saying that because I think these headers are needed by
> p4ofagent, and therefore should be installed. If I'm wrong let me know.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/p4lang/p4-build/pull/5/files/20a1a6c8e8e03e51a76b87fa3a60fbaa0fa1cfc9#r67384920>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFgBcW_QZxssxHmQdwGBefSA3NcwpbKlks5qMYM6gaJpZM4Iylfb>
> .
>